server/src/dbt_execution/DbtRpcClient.ts

```diff
@@ -136,6 +136,12 @@ export class DbtRpcClient {
     await this.makePostRequest<never>(data);
   }
 
+  async deps(): Promise<void> {
```

Comment:
We actually refresh, but looks like it doesn't make sense for `deps` command. Sometimes we can't install packages if they are not compatible between each other. To solve this issue we should somehow show output of `dbt deps`. It can be separated task I think. In the context of this task I added code to start `dbt deps` if server started without dbt packages installed.